lib/taxonomy/topic_taxonomy.rb

```diff
@@ -37,11 +37,15 @@ def ordered_branches
 
     def transform_taxon(taxons, selected_taxon_content_ids)
       taxons.map do |taxon|
+        checked = taxon.taxon_list.any? do |descendant_taxon|
+          selected_taxon_content_ids.include?(descendant_taxon.content_id)
+        end
+
         {
           label: taxon.name,
           value: taxon.content_id,
           items: (transform_taxon(taxon.children, selected_taxon_content_ids) if taxon.children),
-          checked: selected_taxon_content_ids.include?(taxon.content_id),
+          checked:,
```

Comment:
is this correct?

Reply:
```suggestion
          checked: checked,
```

should it be this?

Reply:
Yes – so, initially this changed from:

```diff
- checked: selected_taxon_content_ids.include?(taxon.content_id),
+ checked: checked,
```

But now Whitehall is on Ruby 3.1, we can use [shorthand hash syntax](https://dev.to/baweaver/ruby-3-1-shorthand-hash-syntax-first-impressions-19op).

So `{ checked: checked }` becomes `{ checked: }`

And Rubocop actually enforces this new style – so we have to use it.

Reply:
That's not very readable...